includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1074,7 +1074,25 @@ protected function get_wizard_in_cart_payment_gateways() {
 				'name'        => __( 'PayPal Express Checkout', 'woocommerce' ),
 				'image'       => WC()->plugin_url() . '/assets/images/paypal.png',
 				'description' => $paypal_ec_description,
+				'class'       => 'checked',
```

Comment:
I copied it from the stripe value but `checked` seems to just mean that settings should be open if the item is enabled. Would `settings-if-enabled` (or something) be more helpful? (cc @jeffstieler)